tox.ini

```diff
@@ -16,6 +16,7 @@ commands = flake8 kmip/
 [testenv:integration]
 # Note: This requires local or remote access to a KMIP appliance or service
 deps = {[testenv]deps}
+basepython=python2.7
```

Comment:
I think this is good for now since Barbican is using Python 2.7 for their functional tests. We will have to monitor this to see when we should change it.